easybuild/framework/easyconfig/easyconfig.py

```diff
@@ -79,6 +79,8 @@
 # set of configure/build/install options that can be provided as lists for an iterated build
 ITERATE_OPTIONS = ['preconfigopts', 'configopts', 'prebuildopts', 'buildopts', 'preinstallopts', 'installopts']
 
+PRESERVE_TEMPLATE_REGEX = re.compile(r'(%)(?!%*\(\w+\)s)')
```

Comment:
sure there is, compiling takes time too... That's the whole point of precompiling a regex, in case you'll be using it over and over again.